packages/textarea/src/stories/Textarea.tw.stories.ts

```diff
@@ -0,0 +1,61 @@
+import { action } from '@storybook/addon-actions'
```

Comment:
you can remove this file, since tailwind stories cannot be defined until Lucia defines how the textarea will look